lantern/relational_database/homework.py

```diff
@@ -126,7 +146,8 @@ def task_10_list_first_10_customers(cur):
 
     Results: 10 records
     """
-    pass
+    cur.execute("Select * FROM Customers LIMIT 10")
```

Comment:
format your code everything in one way, above you wrote `SELECT`, here 'Select'...